src/web/app/components/question-edit-form/feedback-path.pipe.spec.ts

```diff
@@ -1,15 +1,90 @@
+import { FeedbackParticipantType } from '../../../types/api-output';
 import { GiverTypeDescriptionPipe, RecipientTypeDescriptionPipe } from './feedback-path.pipe';
 
 describe('GiverTypeDescriptionPipe', () => {
+  const pipe: GiverTypeDescriptionPipe = new GiverTypeDescriptionPipe();
+
   it('create an instance', () => {
-    const pipe: GiverTypeDescriptionPipe = new GiverTypeDescriptionPipe();
     expect(pipe).toBeTruthy();
   });
+
+  it('transform FeedbackParticipantType.SELF', () => {
+    expect(pipe.transform(FeedbackParticipantType.SELF))
+      .toBe('Feedback session creator (i.e., me)');
+  });
+
+  it('transform FeedbackParticipantType.STUDENTS', () => {
+    expect(pipe.transform(FeedbackParticipantType.STUDENTS))
+      .toBe('Students in this course');
+  });
+
+  it('transform FeedbackParticipantType.INSTRUCTORS', () => {
+    expect(pipe.transform(FeedbackParticipantType.INSTRUCTORS))
+      .toBe('Instructors in this course');
+  });
+
+  it('transform FeedbackParticipantType.TEAMS', () => {
+    expect(pipe.transform(FeedbackParticipantType.TEAMS))
+      .toBe('Teams in this course');
+  });
+
+  it('transform default', () => {
+    expect(pipe.transform('' as FeedbackParticipantType))
+      .toBe('Unknown');
```

Comment:
Let's test an existing (but invalid) type too and make sure it works.
For example, `FeedbackParticipantType.OWN_TEAM` should return `Unknown` instead of `Giver's team`